includes/reader-revenue/my-account/class-woocommerce-my-account.php

```diff
@@ -514,6 +522,128 @@ public static function show_message_after_logout() {
 			WooCommerce_Connection::add_wc_notice( __( 'You have successfully logged out.', 'newspack-plugin' ), 'success' );
 		}
 	}
+
+	/**
+	 * Adds option to display memberships without subscriptions on the Subscriptions tab of My Account..
+	 *
+	 * @param array $settings WooCommerce Memberships settings.
+	 * @return array
+	 */
+	public static function option_display_memberships_without_subs( $settings ) {
+		$settings[] = array(
```

Comment:
Very very minor nitpick, but as a team we tend to prefer the shorthand array syntax instead of the "classic" `array()` function for creating new arrays. So this would look like this instead:

```php
		$settings[] = [
			'name' => __( 'Newspack Reader Activation', 'newspack-plugin' ),
			'type' => 'title',
			'id'   => 'wc_memberships_show_unsubbed_memberships',
		];
```

Neither is better than the other, but I think we just decided that `[]` is more slightly more readable at a glance than `array()`.

Reply:
Ah nice! Thanks @dkoo -- consistency is always good 🙂 I updated this, but then ended up deleting it for the new React option.